src/krb5/_creds_mit.pyx

```diff
@@ -120,3 +151,55 @@ EtypeInfo = collections.namedtuple('EtypeInfo', [
     'salt',
     's2kparams',
 ])
+
+def serialize_creds(Context context not None, Creds self not None) -> bytes:
```

Comment:
This should reflect the underlying krb5 API name without the `krb5_` prefix, e.g. `marshal_credentials` and `unmarshal_credentials`.

Might also be better to use another name instead of `self` here as it might confuse people as to why this isn't a class method.